### DISCUSSION TITLE
Column Actor/actrees in Castlist Show Integer 0 on movie dialog - Moviedatabase

### DISCUSSION THREAD

**Discussion by ahmadfauzanarif** [serenity-is/Serenity#5996] [Help]
Hello, I want to ask. I followed the MovieDatabase tutorial and experienced an error in The cast and characters they played, here I use DBMS Mysql. Why column actor/actrees displays the number 0. Is there a solution?

![poto gabisa](https://user-images.githubusercontent.com/71299117/134476694-4f418ea9-4bba-4e5a-adf7-8989711586ea.png)

**Comment by luongnv6** [comment]
please show your code

**Reply by ahmadfauzanarif** [reply]
MovieCastEditDialog.ts


namespace ProjectSerenity.MovieDB {

    @Serenity.Decorators.registerClass()
    export class MovieCastEditDialog extends
        Serenity.Extensions.GridEditorDialog<MovieCastRow> {
        protected getFormKey() { return MovieCastForm.formKey; }
        protected getNameProperty() { return MovieCastRow.nameProperty; }
        protected getLocalTextPrefix() { return MovieCastRow.localTextPrefix; }

        protected form: MovieCastForm;

        constructor() {
            super();
            this.form = new MovieCastForm(this.idPrefix);
        }
    }
}


----------------------------------
MovieCastEditor.ts


namespace ProjectSerenity.MovieDB {
    @Serenity.Decorators.registerEditor()
    export class MovieCastEditor
        extends Serenity.Extensions.GridEditorBase<MovieCastRow> {
        protected getColumnsKey() { return "MovieDB.MovieCast"; }
        protected getDialogType() { return MovieCastEditDialog; }
        protected getLocalTextPrefix() { return MovieCastRow.localTextPrefix; }

        constructor(container: JQuery) {
            super(container);
        }

        protected getAddButtonCaption() {
            return "Add";
        }

        protected validateEntity(row: MovieCastRow, id: number) {
            if (!super.validateEntity(row, id))
                return false;

            row.PersonFullname = PersonRow.getLookup()
                .itemById[row.PersonId].Fullname;

            return true;
        }
    }
}
